cdc/entry/mounter.go

```diff
@@ -112,6 +111,10 @@ func (m *mounterImpl) DecodeEvent(ctx context.Context, pEvent *model.Polymorphic
 	pEvent.Row = rowEvent
 	pEvent.RawKV.Value = nil
 	pEvent.RawKV.OldValue = nil
+	pEvent.Row.Filtered = m.filter.ShouldIgnoreDMLEvent(
```

Comment:
Perhaps we can avoid adding fields to the`RowChangedEvent` by returning parameters from the `Decode` function. For example:
```go
func (m *mounterImpl) DecodeEvent(ctx context.Context, pEvent *model.PolymorphicEvent) (row *model.PolymorphicEvent, err error)
```
In sorter: 
```go
newRow, err := mounter.DecodeEvent(ctx, oldRow)
if newRow == nil {
    continue
}
```

Reply:
done, PTAL